src/site_configs/picknik_kinova_satellite_sim/config/config.yaml

```
@@ -0,0 +1,64 @@
+###############################################################
+#
+#  This configures the robot to work with MoveIt Pro
+#
+###############################################################
+
+# Extend the basic Kinova MuJoCo configuration
+based_on_package: kinova_gen3_mujoco_config
+
+# Baseline hardware configuration parameters for MoveIt Pro.
+# [Required]
+hardware:
+  # Parameters used to configure the robot description through XACRO.
+  # A URDF and SRDF are both required.
+  # [Required]
+  robot_description:
+    urdf:
+      package: "picknik_kinova_satellite_sim"
+      path: "description/picknik_kinova_satellite_sim.xacro"
+    srdf:
+      package: "picknik_kinova_satellite_sim"
+      path: "config/moveit/picknik_kinova_satellite_sim.srdf"
+    # Specify any additional parameters required for the URDF.
+    # Many of these are specific to the UR descriptions packages, and can be customized as needed.
+    # [Optional]
+    urdf_params:
+      - arm: "gen3"
+      - dof: "7"
+      - mujoco_keyframe: "view_tag"
+      - mujoco_model: "description/mujoco/scene.xml"
+      - mujoco_viewer: "true"
+
+# Configuration for launching ros2_control processes.
+# [Required, if using ros2_control]
+ros2_control:
+  # MoveIt Pro will load and activate these controllers at start up to ensure they are available.
+  # If not specified, it is up to the user to ensure the appropriate controllers are active and available
+  # for running the application.
+  # [Optional, default=[]]
+  controllers_active_at_startup:
+    - "force_torque_sensor_broadcaster"
+    - "robotiq_gripper_controller"
+    - "joint_state_broadcaster"
+    - "servo_controller"
+  # Load but do not start these controllers so they can be activated later if needed.
+  controllers_inactive_at_startup:
+    - "joint_trajectory_controller"
+    - "joint_trajectory_admittance_controller"
+    - "velocity_force_controller"
+
+# Configuration for loading behaviors and objectives.
+# [Required]
+objectives:
+  # Specify source folder for objectives
+  # [Required]
+  objective_library_paths:
+    core:
```

```suggestion
    satellite_sim_objectives:
```
The config we inherit from here "kinova_gen3_mujoco_config" also keeps it's objective library path under core so those get overwritten here. If you change the name then we can inherit from it and you can remove most of the objectives in the objectives directory in this config to lower our maintenance burden.

You probably need to keep at least one so the objectives directory still gets installed, so I'd recommend removing all of the objectives except a simple one like clear snapshot.

---

A ha, gotcha. Will create a quick ticket for this and the submodule fix, but will merge for now as per your comment below